src/main/java/me/botsko/prism/utils/TypeUtils.java

```diff
@@ -1,185 +1,181 @@
+                buffer.append(delimiter);
+        }
+        return buffer.toString();
+    }
 
-	/**
-	 * Method to join array elements of type string
-	 * 
-	 * @author Hendrik Will, imwill.com, bug fixes by viveleroi
-	 * @param inputArray Array which contains strings
-	 * @param glueString String between each array element
-	 * @return String containing all array elements separated by glue string
-	 */
-	public static String join(String[] inputArray, String glueString) {
-		String output = "";
-		if (inputArray.length > 0) {
-			StringBuilder sb = new StringBuilder();
-			if (!inputArray[0].isEmpty()) {
-				sb.append(inputArray[0]);
-			}
-			for (int i = 1; i < inputArray.length; i++) {
-				if (!inputArray[i].isEmpty()) {
-					if (sb.length() > 0) {
-						sb.append(glueString);
-					}
-					sb.append(inputArray[i]);
-				}
-			}
-			output = sb.toString();
-		}
-		return output;
-	}
+    /**
+     * Method to join array elements of type string
+     *
+     * @param inputArray Array which contains strings
+     * @param glueString String between each array element
+     * @return String containing all array elements separated by glue string
+     * @author Hendrik Will, imwill.com, bug fixes by viveleroi
+     */
+    public static String join(String[] inputArray, String glueString) {
+        String output = "";
+        if (inputArray.length > 0) {
+            StringBuilder sb = new StringBuilder();
+            if (!inputArray[0].isEmpty()) {
+                sb.append(inputArray[0]);
+            }
+            for (int i = 1; i < inputArray.length; i++) {
+                if (!inputArray[i].isEmpty()) {
+                    if (sb.length() > 0) {
+                        sb.append(glueString);
+                    }
+                    sb.append(inputArray[i]);
+                }
+            }
+            output = sb.toString();
+        }
+        return output;
+    }
 
-	/**
-	 * 
-	 * @param s
-	 * @return
-	 */
-	public static String strToUpper(String s) {
-		return s.substring(0, 1).toUpperCase() + s.substring(1).toLowerCase();
-	}
+    /**
+     * @param s
+     * @return
+     */
+    public static String strToUpper(String s) {
+        return s.substring(0, 1).toUpperCase() + s.substring(1).toLowerCase();
+    }
 
-	/**
-	 * Java implementation of preg_match_all by https://github.com/raimonbosch
-	 * 
-	 * @param p
-	 * @param subject
-	 * @return
-	 */
-	public static String[] preg_match_all(Pattern p, String subject) {
-		Matcher m = p.matcher(subject);
-		StringBuilder out = new StringBuilder();
-		boolean split = false;
-		while (m.find()) {
-			out.append(m.group());
-			out.append("~");
-			split = true;
-		}
-		return (split) ? out.toString().split("~") : new String[0];
-	}
+    /**
+     * Java implementation of preg_match_all by https://github.com/raimonbosch
+     *
+     * @param p
+     * @param subject
+     * @return
+     */
+    public static String[] preg_match_all(Pattern p, CharSequence subject) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'preg_match_all' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)